app/models/post.rb

```diff
@@ -118,7 +118,11 @@ def path
     if title
       prefix = (author.has_custom_domain) ? "" : "/#{author.url_segment}" + (author.username? ? "" : "/posts")
       if page
-        "#{prefix}/#{title.parameterize}"
+        if title.match(PostsHelper::POST_TITLE_NO_SYMBOLS_PATTERN)
+          "#{prefix}/#{title.parameterize}"
+        else
+          "#{prefix}/#{CGI.escape(title)}"
```

Comment:
Do we not also want to parameterize the title here? And why not just CGI escape always and not conditionally?

Reply:
parameterize eliminates all the symbols, that's why there can't be pages with symbols.
I _could_ use always CGI escape but when I have to "unescape it" I also have to consider the links that were generated before this PR (to avoid breaking them). Even if I always use CGI.escape here, I'll have to continue "escaping" in two different ways: https://github.com/standardnotes/listed/pull/168/files#diff-41547fca0ee5b1e4a7a2be25356b71d2fc7389aa1e86fe97fc10cd6f48d1c6dbR27 
Should I go with that option? "escape" the same way and "unescape" in two ways?

Reply:
Ah ok, no need then.